docs-2.0/3.ngql-guide/14.native-index-statements/2.1.show-create-index.md

```diff
@@ -32,22 +30,25 @@ nebula> SHOW CREATE TAG INDEX player_index_1;
 +------------------+--------------------------------------------------+
 ```
 
-Edge indexes can be queried through a similar approach:
+Edge indexes can be queried through a similar approach.
 
 ```ngql
 nebula> SHOW EDGE INDEXES;
-+----------------+
-| Names          |
-+----------------+
-| "index_follow" |
-+----------------+
++----------------+----------+---------+
+| Index Name     | By Edge  | Columns |
++----------------+----------+---------+
+| "follow_index" | "follow" | []      |
++----------------+----------+---------+
 
-nebula> SHOW CREATE EDGE INDEX index index_follow;
+nebula> SHOW CREATE EDGE INDEX follow_index;
 +-----------------+-------------------------------------------------+
 | Edge Index Name | Create Edge Index                               |
 +-----------------+-------------------------------------------------+
-| "index_follow"  | "CREATE EDGE INDEX `index_follow` ON `follow` ( |
-|                 |  `degree`                                       |
+| "follow_index"  | "CREATE EDGE INDEX `follow_index` ON `follow` ( |
 |                 | )"                                              |
 +-----------------+-------------------------------------------------+
 ```
+
+!!! Compatibility "Legacy version compatibility"
+
+    In Nebula Graph 2.0.1, the `SHOW TAG/EDGE INDEXES` statement only returns `Names`.
```

Comment:
2.0.1
is it macro? or hardcode

Reply:
Hardcode. And in 2.0.1, this statement only returns names.